assets/stylesheets/shared/_colors.scss

```diff
@@ -1,74 +1,77 @@
+/**
+ * Colors
+ *
+ * Color variables based on the design handbook
+ * See wordpress.com/design-handbook/colors/ for more info.
+ */
+
 // Primary Accent (Blues)
 $blue-wordpress:         #0087be;
 $blue-light:             #78dcfa;
 $blue-medium:            #00aadc;
 $blue-dark:              #005082;
 
-// Jetpack Green
-$green-jetpack:          #8cc258;
-
 // Grays
 $gray:                   #87a6bc;
 $gray-light:             lighten( $gray, 33% ); //#f3f6f8
+$gray-lighten-30:        lighten( $gray, 30% ); //#e9eff3
+$gray-lighten-20:        lighten( $gray, 20% ); //#c8d7e1
+$gray-lighten-10:        lighten( $gray, 10% ); //#a8bece
+$gray-darken-10:         darken( $gray, 10% ); //#668eaa
+$gray-darken-20:         darken( $gray, 20% ); //#4f748e
+$gray-darken-30:         darken( $gray, 30% ); //#3d596d
```

Comment:
First of all thanks a lot for your feedback, @folletto!

You're right, I'd agree that listing the variables like this doesn't necessarily add clarity. What I'd argue though is that it does help to prevent the ab(use) of color functions like lighten/darken. 

Let me give an example: While working on this feature, I discovered seven different variations of a single color via color functions in a single component (4%, 5%, 10%, 12%, 13%, 17%, 100% of Wordpress Blue). This defeats the color consolidation efforts behind using color variables, compiled it's essentially seven different colors. Color functions like lighten/darken make it very easy to use and create colors that are off-brand.  

Thus in order to discourage the use of color functions, my idea here was the following: we could explicitly declare all colors in the WordPress.com design handbook in the colors file and encourage the explicit use of the color variables as defined in this file only. These should cover most cases of blues and grays. If further darker/lighter shades of other colors are needed those too should be added to the colors file and then used explicitly. This way we can easily keep track of all (shades of) colors used in one place and ensure that we're staying on-brand.

This was the rationale behind adding these explicit color variables. Would be great to get your thoughts on this. Does this sound like a feasible way to reduce color variations to you?